lesson_02/job2_execute.py

```diff
@@ -0,0 +1,14 @@
+from job2.dl.extract import get_extract
+from job2.bl.transform import get_transform_result
+from job2.dl.load import load_stg
+
+def main():
+    period='2022-08-09'
```

Comment:
this parameter not need to hardcode, if we need to use other pages or date. The one date can have data for several pages

Reply:
Hardcoded because I dont use Flask. Sorry so time for it - becauese i need to catch group and make homework intime in future